docs/reference/contrib/modeladmin/primer.rst

```diff
@@ -142,15 +142,20 @@ Overriding templates
 --------------------
 
 For all modeladmin views, Wagtail looks for templates in the following folders
-within your project, before resorting to the defaults:
+within your project before resorting to the defaults:
 
 1. ``/modeladmin/app-name/model-name/``
 2. ``/modeladmin/app-name/``
 3. ``/modeladmin/``
 
-So, to override the template used by ``IndexView`` for example, you'd create a
-new ``index.html`` template and put it in one of those locations.  For example,
-if you wanted to do this for an ``ArticlePage`` model in a ``news`` app, you'd
+As when you `customise the Wagtail admin templates <https://docs.wagtail.io/en/v2.4/advanced_topics/customisation/admin_templates.html?highlight=wagtailadmin#customising-admin-templates>`_, you can create a ``/templates/`` folder within one of your installed apps in order to place the ``/modeladmin/`` folder.  If, for example, you create a ``dashboard`` app for the purpose of containing all logic related to customising the Wagtail admin, then you could set up the following overall folder structure:
```

Comment:
Can this please be updated to use the RST ref system, otherwise this link will load 'old' versions of the docs on future versions using this link.

eg.
```rst
As when you :ref:`customise the Wagtail admin templates <advanced_customising_admin_templates>`, you can create a...
```

Then on the file `docs/advanced_topics/customisation/admin_templates.rst`
add the following *two* lines to the top of the file.

```rst
.. _advanced_customising_admin_templates:

```

Note that the 'ref' links to the file via the `.. _some_name:` syntax. but the ref removes the leading underscore, finally this will not work correctly if you skip the line break after the location declaration at the top of the admin_templates file.

Reply:
I would also add that we might be making too big a jump here conceptually. I feel that it might be better to restore the previous content, but instead of providing just one example, show all three of the previous locations (prefixed with `templates/`) with `index.html` at the end.

Then, we can tackle the issue of "where in a project to create template folders" separately. And whatever we do here, we should provide a caveat that Django's customisable nature means that templates can technically live anywhere in a project, but common approaches include:

1. Adding a `templates/` folder to the relevant app.
2. Adding them to the project's root `templates/` folder.
3. Creating an app for the explicit purpose of overriding default Django/Wagtail behaviour, and adding override templates to a `templates/` folder in that app.

And provide a simple directory structure example for each of these, so as not to show bias for one approach or the other.

Reply:
Thinking on this some more , I think we CAN safely recommend that app-specific templates are better off being placed in app-specific folders, because it improves app portability, and would make the templates easier to find, should things need tweaking in future. 

Options 2 and 3 are only really a better options for project-wide overrides, which I would imagine aren't as common.